wp-modules/editor/js/src/index.ts

```diff
@@ -25,11 +25,6 @@ addFilter(
 	'pattern-manager/preventTransform',
 	preventTransform
 );
-addFilter(
-	'blocks.registerBlockType',
-	'pattern-manager/registerPatternBlock',
-	registerPatternBlock
-);
```

Comment:
Moving `addFilter()` out of this file is the only part of this PR I don't like. 

The idea of this file was to run all of the side-effects of the `editor/` module.

So importing another file shouldn't run side-effects.

Reply:
Still, it's just my opinion 😄

Reply:
Yeah I can see the argument for both ways. My preference is to be able to remove the whole block by simply deleting a directory. But I'd be alright with putting it back.

Reply:
Sounds good, it's a minor issue 😄 Whatever you'd like.

Reply:
I moved it back :)

Reply:
Thanks a lot!